packages/core-bridge/index.d.ts

```diff
@@ -128,15 +128,20 @@ export interface WorkerOptions {
   maxCachedWorkflows: number;
 }
 
-export interface CoreLog {
+/** Log level - must match rust log level names */
+export type LogLevel = 'TRACE' | 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
+
+export interface LogEntry {
   /** Log message */
   message: string;
-  /** Unix millis since epoch. Ideally would be bigint but Neon can't send those
+  /**
+   * Time since epoch [seconds, nanos].
```

Comment:
Why not just stick with millis? Nanos is pretty pointless for log messages and the single number is much easier to handle.

Reply:
I switched because we need higher time resolution for merging the node / core logs in order.